src/fancy_dropdowns.js

```diff
@@ -5,14 +5,29 @@ $(document).ready(function() {
 
 	// Functions
 	var defaults = function(dest,source) {
-		for(var k in source) {
-			if(dest[k] === void 0) {
-				dest[k] = source[k];
-			}
-		}
+				for(var k in source) {
+					if(dest[k] === void 0) {
+						dest[k] = source[k];
+					}
+				}
 
-		return dest;
-	};
+				return dest;
+			},
+
+		  markupEscapeCodes = {
```

Comment:
Sorry to nitpick, but looks like this is using a separate indent character? Is it possible to have it automatically match what currently exists (tab or 2 spaces, don't actually remember).

![image](https://cloud.githubusercontent.com/assets/1239145/6158778/be2dc662-b202-11e4-978f-b59f64d2be3f.png)